packages/phone-sensors/internal/provider/provider.ts

```diff
@@ -0,0 +1,55 @@
+import { PushProvider } from '@awarns/core/providers';
+import { TriAxialType } from '../tri-axial';
+import { Utils } from '@nativescript/core';
+import { PhoneSensor, toNativeSensor } from '../phone-sensor';
+import { getTriAxialReceiver } from '../receiver';
+import { defaultConfig, providerConfigToNativeConfig, ProviderConfiguration } from './provider-configuration';
+
+import ServiceManager = es.uji.geotec.backgroundsensors.service.manager.ServiceManager;
+import BaseSensorRecordingService = es.uji.geotec.backgroundsensors.service.BaseSensorRecordingService;
+import SensorManager = es.uji.geotec.backgroundsensors.sensor.SensorManager;
+import CollectionConfiguration = es.uji.geotec.backgroundsensors.collection.CollectionConfiguration;
+import BaseSensor = es.uji.geotec.backgroundsensors.sensor.BaseSensor;
+import RecordCallback = es.uji.geotec.backgroundsensors.record.callback.RecordCallback;
+import TriAxialRecord = es.uji.geotec.backgroundsensors.record.TriAxialRecord;
+
+export class PhoneSensorsProvider implements PushProvider {
+  static callback = new RecordCallback({
+    onRecordsCollected: (records: java.util.List<TriAxialRecord>) => getTriAxialReceiver().onReceive(records),
+  });
```

Comment:
How about a lazy get function rather than an attribute? To avoid creating this on start, which may fail in certain situations (depending on when and where you import this file)